content/id/docs/concepts/cluster-administration/certificates.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Sertifikasi
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Saat menggunakan autentikasi sertifikat klien, kamu dapat membuat sertifikat
+secara manual melalui `easyrsa`, `openssl` atau `cfssl`.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+### easyrsa
+
+**easyrsa** secara manual dapat menghasilkan sertifikat untuk kluster kamu.
+
+1. Unduh, buka paket, dan inisialisasi versi tambal easyrsa3.
+
+        curl -LO https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz
+        tar xzf easy-rsa.tar.gz
+        cd easy-rsa-master/easyrsa3
+        ./easyrsa init-pki
+1. Hasilkan CA. (`--batch` atur mode otomatis. `--req-cn` _default_ CN untuk digunakan.)
+
+        ./easyrsa --batch "--req-cn=${MASTER_IP}@`date +%s`" build-ca nopass
+1. Hasilkan sertifikat dan kunci _server_.
+    Argumen `--subject-alt-name` menetapkan IP yang mungkin dan nama DNS yang akan diakses
+    oleh _server_ API. `MASTER_CLUSTER_IP` biasanya merupakan IP pertama dari layanan CIDR
+    yang ditentukan sebagai argumen` --service-cluster-ip-range` untuk _server_ API dan
+    komponen manajer pengontrol. Argumen `--days` digunakan untuk mengatur jumlah hari setelah
+    sertifikat berakhir.
+    Sampel di bawah ini juga mengasumsikan bahwa kamu menggunakan `cluster.local` sebagai standar
+    nama ranah DNS.
```

Comment:
I prefer to use: nama _domain_ DNS _default_ than translate it to Bahasa Indonesia.